skins/cat17/templates/partials/payment_preset_amounts.html.twig

```diff
@@ -3,9 +3,9 @@
 	<div class="wrap-radio">
 		{# Amount value needs to be multiplied with 100 because the JavaScript uses integer cent amounts #}
 		{# TODO use proper money objects for predefined amounts in templates #}
-		<input type="radio" id="amount{$ loop.index $}" value="{$ amount * 100 $}">
+		<input type="radio" id="amount{$ loop.index $}" value="{$ amount $}">
 		<label for="amount{$ loop.index $}">
-			<span>{$ amount $} €</span>
+			<span>{$ amount / 100 $} €</span>
```

Comment:
We really should not be doing that, no devision in particular.
The cleanest way, in my eyes, would be to pass the amounts into the view as objects (`Euro`), using one of its member functions + a (i18n-aware) function to format it for the view.
Poor man's version for the moment: `{$ amount.getEuroFloat() | number_format( 0 ) $}`
In the future we could build our own filter to format monies.

https://github.com/wmde/Euro/blob/master/src/Euro.php#L115
https://twig.symfony.com/doc/2.x/filters/number_format.html